ic-agent/src/agent/mod.rs

```diff
@@ -421,13 +422,9 @@ impl Agent {
             .await
             .and_then(|response| match response {
                 replica_api::QueryResponse::Replied { reply } => Ok(reply.arg),
-                replica_api::QueryResponse::Rejected {
-                    reject_code,
-                    reject_message,
-                } => Err(AgentError::ReplicaError {
-                    reject_code,
-                    reject_message,
-                }),
+                replica_api::QueryResponse::Rejected(response) => {
```

Comment:
@ericswanson-dfinity Here you see how the logic of mapping rejected query calls to Replica Error stays the same.